modules/cas_cache/layer_cache_management.c

```diff
@@ -180,7 +180,7 @@ static int _cache_mngt_lock_sync(ocf_cache_t cache)
 	if (!context)
 		return -ENOMEM;
 
-	_cache_mngt_async_context_init(context);
```

Comment:
Maybe we could introduce more descriptive pair of `_cache_mngt_async_context_init()` / `_cache_mngt_async_context_reinit()`. To me it looks more consise and informative than additional boolean parameter.

Reply:
I was considering it but I was not sure which one will be better. Your suggestion assures me that adding `_cache_mngt_async_context_reinit()` function is better solution.